meta-ostro/lib/oeqa/selftest/iotsstatetests.py

```diff
@@ -127,7 +127,7 @@ def get_hashes(d, subdir):
                                      stdout=subprocess.PIPE,
                                      stderr=subprocess.STDOUT)
                 stdout, stderr = p.communicate()
-                analysis.append(stdout)
+                analysis.append(stdout.decode("ascii"))
```

Comment:
On Thu, Aug 25, 2016 at 9:26 AM, Patrick Ohly <notifications@github.com>
wrote:

> In meta-ostro/lib/oeqa/selftest/iotsstatetests.py
> <https://github.com/ostroproject/meta-ostro/pull/255#discussion_r76186611>
> :
>
> > @@ -127,7 +127,7 @@ def get_hashes(d, subdir):
> >                                       stdout=subprocess.PIPE,
> >                                       stderr=subprocess.STDOUT)
> >                  stdout, stderr = p.communicate()
> > -                analysis.append(stdout)
> > +                analysis.append(stdout.decode("ascii"))
>
> I prefer decode("utf-8") - a bit more future-proof, but shouldn't matter
> in practice.
>

I thought about it. Will change to utf-8.